test/images/embed.go

```diff
@@ -14,6 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package generated
+package images
 
-// No code is needed here.  This is a stub for compilation purposes.
+import (
+	"embed"
+
+	"k8s.io/kubernetes/test/e2e/framework/testfiles"
+)
+
+//go:embed *
```

Comment:
there's a **lot** of stuff under test/images ... were we building it all into generated bindata before? if not, I don't think we should slurp it all into an embedded file like this

Reply:
Yes. We were generated bindata for all of that (ref: https://github.com/kubernetes/kubernetes/blob/36a19df91baf8d042f8ba13063905c6395737b7e/hack/generate-bindata.sh#L48) barring Markdown, JPG and PNG files.

Reply:
so this expands to include markdown/images/all files? is there a way to avoid that?

Reply:
Right now, the go:embed syntax doesn't provide a way to exclude files from a filesystem where a wildcard expression includes everything.

We have to specifically include files if we want to pick what we want and what we don't. This is like a gotcha. I am not very happy with this limitation.

Reply:
if we can check which files were in use, perhaps inverting to only include what we actually use?

Reply:
I grep'ed through references of the embedded files and trimmed down the number of files that were being embedded.